fedora-spec-files/Makefile

```diff
@@ -0,0 +1,18 @@
+IMAGE_NAME := fedora-spec-research
+PODMAN_RUN_THIS_DIR := podman run --rm -v $(CURDIR):/fedora-spec-files:Z -w /fedora-spec-files
+
+# build the container image with the jupyter notebook and all the dependencies
+build:
+	podman build --tag=$(IMAGE_NAME) .
+
+# run jupyter notebook
+run:
+	@# the rooless user doesn't have access to the folder
+	@chmod -R +rwx .
```

Comment:
Things should work without this. The "rooless" user has access to all the volume mounts done with the `Z` flag.

Reply:
mine didn't, I had to chmod

Reply:
Well, in order to execute `./generate-data.py` directly, the executable flag is indeed needed on it. But `u+x` on that single file and checking it in that way is enough. There is no need to make all the files in the directory executable and writable by anyone. Alternatively, you could run the script [through `python3`](https://github.com/csomh/research/commit/e63be32f3b341d881db1167b5c117ced197923c8).

Reply:
you're right! the perm problem was about the script not having the 'x' bit; I wish "Permission denied" was more descriptive